camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/common/EtlCounts.java

```diff
@@ -65,6 +66,11 @@ public EtlCounts(String topic, long granularity) {
     this(topic, granularity, System.currentTimeMillis());
   }
 
+  public EtlCounts(EtlCounts other) {
```

Comment:
Added copy constructor for the purpose of converting EtlCounts object to either EtlCounts object or EtlCountsForUnitTest object (line 503 in CamusJob.java).